add-new-exercise.ps1

```diff
@@ -36,30 +36,34 @@ param (
 # Import shared functionality
 . ./shared.ps1
 
-$exerciseName = (Get-Culture).TextInfo.ToTitleCase($Exercise).Replace("-", "")
-$exercisesDir = Resolve-Path "exercises"
-$exerciseDir = Join-Path $exercisesDir $Exercise
-
 function Add-Project {
+    param (
+        [Parameter(Position = 0, Mandatory = $true)][string]$Exercise,
+        [Parameter(Position = 1, Mandatory = $true)][string]$ExerciseName
+    )
+
     Write-Output "Adding project"
 
-    $csProj = "$exerciseDir/$exerciseName.csproj"
+    $exercisesDir = Resolve-Path "exercises"
+    $exerciseDir = Join-Path $exercisesDir $Exercise
 
-    Run-Command "dotnet new xunit -lang ""C#"" --target-framework-override netcoreapp3.0 -o $exerciseDir -n $exerciseName"
-    Run-Command "dotnet sln ""$exercisesDir/Exercises.sln"" add $csProj"
+    $csProj = "$exerciseDir/$ExerciseName.csproj"
+
+    Invoke-CommandExecution "dotnet new xunit -lang ""C#"" --target-framework-override net5.0 -o $exerciseDir -n $ExerciseName"
+    Invoke-CommandExecution "dotnet sln ""$exercisesDir/Exercises.sln"" add $csProj"
     
     Remove-Item -Path "$exerciseDir/UnitTest1.cs"
     
-    New-Item -ItemType File -Path "$exerciseDir/$exerciseName.cs"
+    New-Item -ItemType File -Path "$exerciseDir/$ExerciseName.cs"
     New-Item -ItemType File -Path "$exerciseDir/Example.cs"
     
     [xml]$proj = Get-Content $csProj
     $propertyGroup = $proj.Project.PropertyGroup
 
     $nugetItemGroup = $proj.Project.ItemGroup;
     $nugetItemGroup.RemoveAll();
-    $nugetList = @(@{nuget="Microsoft.NET.Test.Sdk";Version="16.7.1"}, @{nuget="xunit";Version="2.4.1"}, @{nuget="xunit.runner.visualstudio";version="2.4.3"})
-    $nugetList | % { 
+    $nugetList = @(@{nuget = "Microsoft.NET.Test.Sdk"; Version = "16.8.3" }, @{nuget = "xunit"; Version = "2.4.1" }, @{nuget = "xunit.runner.visualstudio"; version = "2.4.3" })
```

Comment:
update the version to match the current exercises

Reply:
Excellent!